src/clr-core/icon-shapes/icon.element.ts

```diff
@@ -160,7 +160,9 @@ export class CdsIcon extends IconMixinClass {
 
   @query('svg') private svg: SVGElement;
 
-  private ariaLabel = `aria-${this._idPrefix}${this._uniqueId}`;
+  // TODO: MAKE aria-label A REQUIRED PROPERTY WHEN THAT IS READY
+
+  private ariaLabelId = 'aria-' + this._idPrefix + this._uniqueId;
```

Comment:
I thought we picked the other name: `idForAriaLabel`. 😅 

Let me know if you want to update or if you want me to update my PR to match this name.

Reply:
I’ll update it. Thanks for the heads up!

Reply:
Done.